docs/data-routing/5-examples/24-supabase.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Supabase
+---
+
+import Pipeline from '@site/src/components/usethispipeline'
+
+[Supabase](https://supabase.com/) is an open-source backend-as-a-service
+platform designed to simplify the development of web and mobile applications. It
+provides developers with a suite of tools including a Postgres database,
+real-time subscriptions, authentication, storage, and serverless functions, all
+managed through a single interface. Supabase provides multiple ways to consume
+data from Golioth.
+
+The first example is the simplest and uses the REST API. Assuming a database
+named `golioth_pipeline_basic` with the following columns:
+
+| id | created_at | temp | lat | long |
+| -- | ---------- | ---- | --- | ---- |
+
+Create the following Pipeline:
+
+<Pipeline link='https://console.golioth.dev/pipeline?name=Supabase%20Edge%20Function%20Example&pipeline=c3RlcHM6CiAgLSBuYW1lOiBzdGVwMAogICAgZGVzdGluYXRpb246CiAgICAgIHR5cGU6IHdlYmhvb2sKICAgICAgdmVyc2lvbjogdjEKICAgICAgcGFyYW1ldGVyczoKICAgICAgICB1cmw6IGh0dHBzOi8vcmdnY3Fvc2x0cW9wY3d0d3doa2Muc3VwYWJhc2UuY28vZnVuY3Rpb25zL3YxL2dvbGlvdGgtcGlwZWxpbmUKICAgICAgICBoZWFkZXJzOgogICAgICAgICAgQXV0aG9yaXphdGlvbjogJFNVUEFCQVNFX0FVVEhfSEVBREVSCiAgICAgICAgICBhcGlrZXk6ICRTVVBBQkFTRV9BUElfS0VZCg=='/>
```

Comment:
```suggestion
<Pipeline link='https://console.golioth.io/pipeline?name=Supabase%20Edge%20Function%20Example&pipeline=c3RlcHM6CiAgLSBuYW1lOiBzdGVwMAogICAgZGVzdGluYXRpb246CiAgICAgIHR5cGU6IHdlYmhvb2sKICAgICAgdmVyc2lvbjogdjEKICAgICAgcGFyYW1ldGVyczoKICAgICAgICB1cmw6IGh0dHBzOi8vcmdnY3Fvc2x0cW9wY3d0d3doa2Muc3VwYWJhc2UuY28vZnVuY3Rpb25zL3YxL2dvbGlvdGgtcGlwZWxpbmUKICAgICAgICBoZWFkZXJzOgogICAgICAgICAgQXV0aG9yaXphdGlvbjogJFNVUEFCQVNFX0FVVEhfSEVBREVSCiAgICAgICAgICBhcGlrZXk6ICRTVVBBQkFTRV9BUElfS0VZCg=='/>
```

Reply:
Also name of this one is "Supabase Edge Function Example", which me might want to switch to "Supabase REST API Example" 👍🏻